docs/other.rst

```diff
@@ -74,3 +74,12 @@ CSS Selectors
     import aloe_webdriver.css
 
 .. automodule:: aloe_webdriver.css
+
+Screenshots
+===========
+
```

Comment:
If you really want this, add that line around the import. On all the projects I'm using this, I haven't had a need to turn it off conditionally; do you think it'll be useful? Especially when nothing else is configured via environment variables?